src/server/page_server.cpp

```diff
@@ -236,6 +253,17 @@ page_server::connection_handler::on_data_page_read_result (const FILEIO_PAGE *io
   m_conn->push (page_to_tran_request::SEND_DATA_PAGE, std::move (message));
 }
 
+void
+page_server::connection_handler::prior_sender_sink_hook (std::string &&message) const
+{
+  assert (message.size () > 0);
+
+  // passive tran server already connected - as assumed by the very existence of this
+  // connection handler instance
+  m_conn->push (page_to_tran_request::SEND_TO_PTS_LOG_PRIOR_LIST, std::move (message));
```

Comment:
Let the code explain itself :)
```suggestion
  assert (m_conn != nullptr);
  m_conn->push (page_to_tran_request::SEND_TO_PTS_LOG_PRIOR_LIST, std::move (message));
```